app/src/main/java/com/microsoft/office365/snippetapp/EmailStories/DeleteEmailAttachmentStory.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See full license at the bottom of this file.
+ */
+
+package com.microsoft.office365.snippetapp.EmailStories;
+
+import com.microsoft.office365.snippetapp.R;
+import com.microsoft.office365.snippetapp.Snippets.EmailSnippets;
+import com.microsoft.office365.snippetapp.helpers.AuthenticationController;
+import com.microsoft.office365.snippetapp.helpers.GlobalValues;
+import com.microsoft.office365.snippetapp.helpers.StoryResultFormatter;
+import com.microsoft.outlookservices.Message;
+
+import java.util.concurrent.ExecutionException;
+
+public class DeleteEmailAttachmentStory extends  BaseEmailUserStory{
+
+    private static final String SENT_NOTICE = "Attachment email sent with subject line:";
+    private static final String STORY_DESCRIPTION = "Deletes an attachment from a draft email message";
+    private static final boolean IS_INLINE = false;
+
+
+    @Override
+    public String execute() {
+        String returnResult = "";
+        try {
+            AuthenticationController
+                    .getInstance()
+                    .setResourceId(
+                            getO365MailResourceId());
+
+            EmailSnippets emailSnippets = new EmailSnippets(
+                    getO365MailClient());
+
+            //1. Add an email to draft folder and store the ID
+            String uniqueGUID = java.util.UUID.randomUUID().toString();
+            String mailSubject = getStringResource(R.string.mail_subject_text) + uniqueGUID;
+
+            //Add a new email to the user's draft folder
+            String emailID = emailSnippets.addDraftMail(GlobalValues.USER_EMAIL,
+                    mailSubject,
+                    getStringResource(R.string.mail_body_text));
+
+            //Add a text file attachment to the mail added to the draft folder
+            emailSnippets.addTextFileAttachmentToMessage(emailID
+                    , getStringResource(R.string.text_attachment_contents)
+                    , getStringResource(R.string.text_attachment_filename)
+                    , IS_INLINE);
+
+            StringBuilder sb = new StringBuilder();
```

Comment:
Could reuse returnResult here and change its type to StringBuilder since it gets returned later. Also can chain the .append() calls together and remove the +.